Arrowgene.Ddon.GameServer/Handler/CraftStartEquipColorChangeHandler.cs

```diff
@@ -42,8 +43,16 @@ public override S2CCraftStartEquipColorChangeRes Handle(GameClient client, C2SCr
 
             if (!string.IsNullOrEmpty(DyeUId))
             {
-                var updateResults = Server.ItemManager.ConsumeItemByUIdFromMultipleStorages(Server, client.Character, ItemManager.BothStorageTypes, DyeUId, 1);
-                updateCharacterItemNtc.UpdateItemList.AddRange(updateResults);
+                try
+                {
+                    var updateResults = Server.ItemManager.ConsumeItemByUIdFromMultipleStorages(Server, client.Character, ItemManager.BothStorageTypes, DyeUId, 1);
+                    updateCharacterItemNtc.UpdateItemList.AddRange(updateResults);
+                }
+                catch (NotEnoughItemsException e)
+                {
+                    Logger.Exception(e);
```

Comment:
why not use the exception which has the error code which is shown in the client?